src/Miso/Style.hs

```diff
@@ -0,0 +1,1524 @@
+-- 'borderTopWidth'
+--
+-- > styles_ [ borderTopWidth .= "value" ]
+--
+borderTopWidth :: MisoString -> Style
+borderTopWidth x = "border-top-width" === x
+-----------------------------------------------------------------------------
+-- 'borderWidth'
+--
+-- > styles_ [ borderWidth .= "value" ]
+--
+borderWidth :: MisoString -> Style
+borderWidth x = "border-width" === x
+-----------------------------------------------------------------------------
+-- 'bottom'
+--
+-- > styles_ [ bottom .= "value" ]
+--
+bottom :: MisoString -> Style
+bottom x = "bottom" === x
+-----------------------------------------------------------------------------
+-- 'boxShadow'
+--
+-- > styles_ [ boxShadow .= "value" ]
+--
+boxShadow :: MisoString -> Style
+boxShadow x = "box-shadow" === x
+-----------------------------------------------------------------------------
+-- 'boxSizing'
+--
+-- > styles_ [ boxSizing .= "value" ]
+--
+boxSizing :: MisoString -> Style
+boxSizing x = "box-sizing" === x
+-----------------------------------------------------------------------------
+-- 'clipPath'
+--
+-- > styles_ [ clipPath .= "value" ]
+--
+clipPath :: MisoString -> Style
+clipPath x = "clip-path" === x
+-----------------------------------------------------------------------------
+-- 'color'
+--
+-- > styles_ [ color .= "value" ]
+--
+color :: Color -> Style
+color x = "color" === renderColor x
+-----------------------------------------------------------------------------
+-- 'columnGap'
+--
+-- > styles_ [ columnGap .= "value" ]
+--
+columnGap :: MisoString -> Style
+columnGap x = "column-gap" === x
+-----------------------------------------------------------------------------
+-- 'cssVariable'
+--
+-- > styles_ [ cssVariable .= "value" ]
+--
+cssVariable :: MisoString -> Style
+cssVariable x = "css-variable" === x
+-----------------------------------------------------------------------------
+-- 'direction'
+--
+-- > styles_ [ direction .= "value" ]
+--
+direction :: MisoString -> Style
+direction x = "direction" === x
+-----------------------------------------------------------------------------
+-- 'display'
+--
+-- > styles_ [ display .= "value" ]
+--
+display :: MisoString -> Style
+display x = "display" === x
+-----------------------------------------------------------------------------
+-- 'filter'
+--
+-- > styles_ [ filter .= "value" ]
+--
+filter :: MisoString -> Style
+filter x = "filter" === x
+-----------------------------------------------------------------------------
+-- 'flexBasis'
+--
+-- > styles_ [ flexBasis .= "value" ]
+--
+flexBasis :: MisoString -> Style
+flexBasis x = "flex-basis" === x
+-----------------------------------------------------------------------------
+-- 'flexDirection'
+--
+-- > styles_ [ flexDirection .= "value" ]
+--
+flexDirection :: MisoString -> Style
+flexDirection x = "flex-direction" === x
+-----------------------------------------------------------------------------
+-- 'flexFlow'
+--
+-- > styles_ [ flexFlow .= "value" ]
```

Comment:
Is this `.=` operator in haddock correct? It looks like something from aeson or lens..

Reply:
No, good catch. Let me find and replace w/ VSCode

Reply:
Also all the examples use `styles_` (plural) which doesn't exist.

Reply:
doh

Reply:
fixed